lib/attr_encrypted.rb

```diff
@@ -277,6 +310,7 @@ module InstanceMethods
     #  @user = User.new('some-secret-key')
     #  @user.decrypt(:email, 'SOME_ENCRYPTED_EMAIL_STRING')
     def decrypt(attribute, encrypted_value)
+      self.class.encrypted_attributes[attribute.to_sym][:operation] = :decrypting
```

Comment:
Do you think the `@encrypted_attributes` hash should be converted into something threadsafe like [`Concurrent::Hash`](https://github.com/ruby-concurrency/concurrent-ruby)? Or maybe not the entire hash but one for each attribute instead if that would perform better?

```ruby
@encrypted_attributes[attribute.to_sym] ||= Concurrent::Hash.new
```

Eventually we'll probably want to find a way to do this in a way where we're not mutating a shared variable and instead have each encrypt/decrypt call bound to their own copy of these options.